lib/build.py

```diff
@@ -167,6 +172,10 @@ def build_arg_get(arg):
    global image_ct
    image_ct = sum(1 for i in tree.children_("from_"))
 
+   # Exit if multistage target is specified for single stage Dockerfile.
+   if (cli.target is not None and image_ct < 2):
```

Comment:
I think this would be more clear because "1" matches "single":
```suggestion
   if (cli.target is not None and image_ct <= 1):
```